services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -100,6 +101,7 @@ class LinkSpec extends BaseSpecification {
         }
     }
 
+    @Tags(SMOKE)
```

Comment:
Why not `"Link (not BFD) status is properly changed when link connectivity is broken (not port down)"` ? It is the most _core_ functionality in the whole system, top1 priority to work properly

Reply:
this test was removed from set

and the mentioned test was added